node/pkg/driver/node.go

```diff
@@ -613,9 +627,104 @@ func (d *NodeService) NodeUnpublishVolume(ctx context.Context, req *csi.NodeUnpu
 }
 
 func (d *NodeService) NodeGetVolumeStats(ctx context.Context, req *csi.NodeGetVolumeStatsRequest) (*csi.NodeGetVolumeStatsResponse, error) {
-	goid_info.SetAdditionalIDInfo(req.VolumeId)
+	volumeId := req.VolumeId
+	volumePath := req.VolumePath
+	volumePathWithHostPrefix := d.NodeUtils.GetPodPath(req.VolumePath)
+	isFSVolume := true
+	goid_info.SetAdditionalIDInfo(volumeId)
```

Comment:
please set the additional id info before any other logic (before e.g. `GetPodPath`)

Reply:
also the defer line? or it's doesn't matter?

Reply:
the defer should also come right after the set

Reply:
done